### PR TITLE
Updated dependencies in READMEs

### PR DESCRIPTION
**Pull request by marjoleintamis**
- Readme frontend: added  4 dependencies
- Readme backend: added 7 dependencies, removed quick-xml as it's no longer in cargo.toml
- cargo.toml: added a commented line about updating the README as well

 _What's the scope of the changes?_
Added dependencies to 2 README files, added a comment to cargo.toml

_What did you test? Which edge cases did you explicitly take into account?_
N/A

_Are there things you want reviewers to definitely take a look at?_
Please check if the dependencies are correct and if I did the commenting out alright in the .toml

_Are there specific people you want feedback from?_
@praseodym @oliver3 

_Do you have tips on how to test? (For example: data setup, triggering specific errors, etc)_
N/A

### CODE REVIEW

**Review by oliver3 — APPROVED** [review]
Looked mainly at the frontend dependencies

File: backend/README.md [hunk 1]
```diff
@@ -45,17 +45,23 @@ Use `cargo test` to run the tests. The tests are also run in a GitHub Actions wo
...
-- `tokio`: runtime for writing asynchronous applications.
 - `memory-serve`: serves frontend assets from memory, but ad-hoc from disk during development.
 - `utoipa`: library for documenting REST APIs using OpenAPI.
 - `utoipa-swagger-ui`: Swagger UI for the OpenAPI specification.
 - `serde`: framework for serializing and deserializing data structures.
 - `serde_json`: JSON support for Serde.
 - `sqlx`: async SQL library featuring compile-time checked queries.
-- `chrono`: date and time library.
-- `clap`: library for command-line argument parsing.
-- `quick-xml`: reading and writing EML_NL XML files.
```
Comment (praseodym) [inline backend/README.md:58]: `quick-xml` was removed by accident

Comment (marjoleintamis) [reply backend/README.md:58]: Should it also be added to cargo.toml?

Comment (marjoleintamis) [reply backend/README.md:58]: Oh sorry hadn't seen that PR :)

**Review by praseodym — DISMISSED** [review]

**Review by oliver3 — APPROVED** [review]

**Review by Lionqueen94 — APPROVED** [review]

**Review by cikzh — APPROVED** [review]
Checked the backend dependencies after the merge. Looks good!
